src/ModelTable/ModelTableStoreContext.tsx

```diff
@@ -54,6 +54,7 @@ export interface ModelTableState<
   formOptions?: FormOptions;
   onUpdate?: OnActionTrigger<DT>;
   onDelete?: OnActionTrigger<DT>;
+  idToFocus?: string;
```

Comment:
Rename to `selectedRows: ID[]`; `ID` is a type in conveyor

In the future, we will be able to select multiple rows.

Reply:
Sounds good, implemented!